### ISSUE TITLE
RFE: expel

### ISSUE THREAD

**Issue by jjYBdx4IL** [Dwarf-Therapist/Dwarf-Therapist#230]
Would be nice to have an option to expel (single or multiple) selected dwarves directly from the "Labors Full" tab.

**Comment by jjYBdx4IL** [comment]
Otherwise it's really a pain to handle large migrant waves in a way that does not use the cap configuration setting.

**Comment by cvuchener** [comment]
Has any research been done on how the expulsion process works internally? I tried a quick search on DFHack topic and I did not find anything.  There is the [emigration script](https://github.com/DFHack/scripts/blob/master/emigration.lua), but I understand it is something different.

**Comment by jjYBdx4IL** [comment]
I don't really know anything about the internals. Can one use dfhack to simulate UI interaction, ie. to simply select the in-game "expel" option?

**Comment by lethosor** [comment]
> Can one use dfhack to simulate UI interaction, ie. to simply select the in-game "expel" option?

Definitely, but DT by itself can't do that, since it involves running DF code.
